docs/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "spfx-toolkit-docs",
+  "type": "module",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.1.0",
```
lets make this at least minor first release 🤔
not sure if it makes sense in terms of docs but `0.0.0` just seams strange 😜